### DISCUSSION TITLE
Feature Request: Let Users Select Index in VectorDB

### DISCUSSION THREAD

**Discussion by jeannot-prvlimburg** [microsoft/sample-app-aoai-chatGPT#1082] [Ideas]
I've been working on adding the ability to let users select the index within the VectorDB, but so far, the app still defaults to the index settings it was deployed with, ignoring any new selections.

Could we introduce an option to dynamically choose or change the index type at runtime?

**Comment by jack-vinitsky** [comment]
UPVOTE!  This would be quite useful for us.  Of course then do we also allow the assigning other related settings such as System Prompt, Temperature, Top K to the Index that is switched to?

**Comment by jeannot-prvlimburg** [comment]
I’ve made progress on this functionality, but the current implementation is updating the app settings for all users, instead of just the specific user who made the changes.
